components/admin/test-suite/SPECS/tests.spec

```diff
@@ -8,73 +8,77 @@
 #
 #----------------------------------------------------------------------------eh-
 
+%global __brp_mangle_shebangs_exclude bats
+%undefine __brp_mangle_shebangs
+
 %include %{_sourcedir}/OHPC_macros
 
+%define testuser ohpc-test
+%define homedir %(sed -n "s/^\s*HOME=\s*//p" /etc/default/useradd || echo "/home" )
+
 Summary:   Integration test suite for OpenHPC
 Name:      test-suite%{PROJ_DELIM}
-Version:   2.6.0
+Version:   3.0
 Release:   1
 License:   Apache-2.0
 Group:     %{PROJ_NAME}/admin
 BuildArch: noarch
 URL:       https://github.com/openhpc/ohpc/tests
 Source0:   tests-ohpc.tar
+# In the OHPC source root dir, create with "tar -cf tests-ohpc.tar tests/"
 
 BuildRequires:  autoconf%{PROJ_DELIM}
 BuildRequires:  automake%{PROJ_DELIM}
+BuildRequires:  sed, findutils
 
-%global __brp_mangle_shebangs_exclude bats
-%undefine __brp_mangle_shebangs
-
-%if 0%{?suse_version}
+%if 0%{?suse_version} || 0%{?sle_version}
 Requires(pre):  shadow
-Requires: python-base
+Requires: python3-base
 %endif
 
 %if 0%{?rhel_version}
```

Comment:
Can you also change this to `0%{?rhel}`. This is one of the OBS specific macros which are not available outside of OBS.

Reply:
Good catch. I'll do that.